engines/mtropolis/mtropolis.cpp

```diff
@@ -56,6 +56,7 @@ namespace MTropolis {
 MTropolisEngine::MTropolisEngine(OSystem *syst, const MTropolisGameDescription *gameDesc) : Engine(syst), _gameDescription(gameDesc), _saveWriter(nullptr), _isTriggeredAutosave(false) {
 	const Common::FSNode gameDataDir(ConfMan.getPath("path"));
 	SearchMan.addSubDirectoryMatching(gameDataDir, "Resource");
+	SearchMan.addSubDirectoryMatching(gameDataDir, "Video");
```

Comment:
This code is actually obsolete because of the switch to the virtual file system.  Use a path prefixed with a video directory like the AVI loader currently does, that way this code can properly handle junctions into installer archives and such.

Reply:
Removed